sdk/appcenter-crashes/src/main/java/com/microsoft/appcenter/crashes/Crashes.java

```diff
@@ -836,11 +836,14 @@ public void run() {
                         Map.Entry<UUID, ErrorLogReport> unprocessedEntry = unprocessedIterator.next();
                         ErrorLogReport errorLogReport = unprocessedEntry.getValue();
                         if (errorLogReport.report.getThrowable() instanceof NativeException) {
-                            Exception exception = errorLogReport.log.getException();
-                            dumpFile = new File(exception.getStackTrace());
-                            exception.setStackTrace(null);
```

Comment:
Don't remove setting back to null now that you have the check.
We highly suspect double processing as the customer confirmed it never tried to throw `NativeException`.